lib/capistrano/tasks/slackify.cap

```diff
@@ -4,6 +4,7 @@ namespace :slack do
   task :notify_started do
     run_locally do
       set :time_started, Time.now.to_i
+      set :release_hosts, release_roles(:all).map(&:hostname).join("\n")
```

Comment:
Would it be possible to move this into a default?  Something like:

    set :slack_hosts, -> { release_roles(:all).map(&:hostname).join("\n") }

This would provide an option for those with many hosts to avoid extra noise in Slack.